app/controllers/home_page.coffee

```diff
@@ -24,10 +24,10 @@ class HomePage extends Controller
       el: @el.find '.recents .image-changer'
       sources: []
 
-    # Api.get '/projects/serengeti', (data) =>
-    #   @classificationCount.html data.classification_count
-    #   @userCount.html data.user_count
-    #   @progressFill.width "#{100 * (data.complete_count / @totalSubjects)}%"
+    Api.get '/projects/serengeti', (data) =>
```

Comment:
this is a merge of the already committed change here https://github.com/zooniverse/Serengeti/commit/eff10c05992c4f5b3b06131c282db420cfb1ed75 
Not quite sure why it's showing as a diff